### PR TITLE
Create animals table

### PR DESCRIPTION
**Pull request by dieum512**
In this milestone I:

- Created tables in SQL and
- Insert and query data in SQL.

Here are the pictures I was supposed to send for the results:

Screenshot from 2023-05-24 12-08-50

Screenshot from 2023-05-24 19-03-07
Screenshot from 2023-05-24 19-04-58

Screenshot from 2023-05-24 12-29-00

Screenshot from 2023-05-24 12-32-13

Screenshot from 2023-05-24 12-34-28

Screenshot from 2023-05-24 12-36-43

Screenshot from 2023-05-24 12-39-20

### CODE REVIEW

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
Hi @dieum512 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Please kindly customized every necessary section of your readme 
- Kindly remove all section descriptions
- Kindly fix all broken links
- Kindly update the Tech stack
- Kindly include all necessary details under the `getting started` section to set up your project

image



Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```
Comment (Munsa1) [inline queries.sql]: - [ ] While this query works fine, the time parameters should be between `'2016/01/01' AND date_of_birth <= '2019/31/12'`, kindly fix this

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+SELECT * FROM animals WHERE neutered = true AND escape_attempts < 3;
```
Comment (Munsa1) [inline queries.sql]: - [ ] Again here we are supposed to select the name and not `select * from` as you have done, kindly fix this as well.

requirements screenshot
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @dieum512, 
required-changes-1

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Added ScreenShots To PR Description :heavy_check_mark: 
Correct Git Flow :heavy_check_mark: 
Descriptive PR :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @dieum512, 
congratulations-1

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Made Requested Changes :heavy_check_mark: 
: 

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**